src/components/workflows-cbas/FindWorkflowModal.js

```diff
@@ -0,0 +1,17 @@
+import { h } from 'react-hyperscript-helpers';
+import ModalDrawer from 'src/components/ModalDrawer';
+
+const FindWorkflowModal = ({ onDismiss }) => {
+  return h(
+    ModalDrawer,
+    {
+      'aria-label': 'find-workflow-modal',
```

Comment:
An aria label (if needed) should be regular English, the same as a visual label.